helmfile.d/stacks/monitoring-prometheus.yaml.gotmpl

```
@@ -14,7 +14,7 @@ templates:
     needs:
       - monitoring/networkpolicy
       - monitoring/podsecuritypolicy
-      {{- if .Values | get "clusterApi.enabled" false }}
+      {{- if and (.Values | get "ck8sManagementCluster.enabled" false) (.Values | get "clusterApi.enabled" false) }}
```

Would this work? Will we be in the situation where `clusterApi.enabled: false` and `ck8sManagementCluster.enabled: true`?
```suggestion
      {{- if (.Values | get "ck8sManagementCluster.enabled" false) }}
```

---

For kubespray environments we will have `clusterApi.enabled: false` and `ck8sManagementCluster.enabled: true` so both values need to be checked or it will break our unit tests (see the discussion [here](https://github.com/elastisys/compliantkubernetes-apps/pull/2372#discussion_r1890044266)).